metaStoreApp/app/controllers/api/v1/accounts_controller.rb

```diff
@@ -1,16 +1,89 @@
 module Api
   module V1
     class AccountsController < ApplicationController
+      # displays the data of all the accounts
+      api :GET, '/accounts/', "Fetch all the account details"
+      example %(
+      GET /api/v1/accounts
+      {
+      "status": "SUCCESS",
+      "message": "Loaded accounts",
+      "data": [
+        {
+          "id": 1,
+          "code": "PQR100",
+          "name": "Account1",
+          "password": "Account1",
+          "created_at": "2020-02-25T14:00:01.000Z",
+          "updated_at": "2020-02-25T14:08:55.000Z"
+        },
+        {
+          "id": 2,
+          "code": "XYZ100",
+          "name": "Account2",
+          "password": "Account2",
+          "created_at": "2020-02-25T14:00:19.000Z",
+          "updated_at": "2020-02-25T14:00:19.000Z"
+        }
+      ]
+      }
+      )
       def index
         accounts = Account.all;
         render json: { status: 'SUCCESS', message: 'Loaded accounts', data: accounts }, status: :ok 
       end
 
+      # displays the details of specific account based on the id
+      api :GET, '/accounts/:id' ,"Fetches the specific account based on id"
+      param :id, :number, :desc => "Id"
+      example %(
+      GET /api/v1/accounts/1
+      {
+      "status": "SUCCESS",
+      "message": "Loaded account",
+      "data": {
+        "id": 1,
+        "code": "PQR100",
+        "name": "Account1",
+        "password": "Account1",
+        "created_at": "2020-02-25T14:00:01.000Z",
+        "updated_at": "2020-02-25T14:08:55.000Z"
+      }
+      }
+      )
       def show
         account = Account.find(params[:id])
         render json: { status: 'SUCCESS', message: 'Loaded account', data: account }, status: :ok 
       end
 
+      # creates a new account
+      api :POST, '/accounts/', "Creates a new account"
+      param :code, String, :desc => "Code", :required => true
```

Comment:
use `key`: `value` syntax, use `=>` only if you have ruby object as key.

Reply:
Okay